dev-libs/libpeas/libpeas-1.36.0.ebuild

```diff
@@ -12,7 +12,7 @@ DESCRIPTION="A GObject plugins library"
 HOMEPAGE="https://wiki.gnome.org/Projects/Libpeas https://gitlab.gnome.org/GNOME/libpeas"
 
 LICENSE="LGPL-2.1+"
-SLOT="0"
+SLOT="1"
```

Comment:
We can't do this without a "slotmove". See https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=7f11661f41a58cf8fb860ea23db39322c608ee7e for example.

But IMO there's not really any value in changing the slot.

The thing we definitely do need to do is updating reverse dependencies of libpeas to depend on a specific slot. Otherwise when we add the new slot, those packages will likely fail to build if portage chooses to provide only the new libpeas version.